docs/en/Text-Templating-Razor.md

```diff
@@ -84,6 +84,7 @@ public class DemoTemplateDefinitionProvider : TemplateDefinitionProvider
     {
         context.Add(
             new TemplateDefinition("Hello") //template name: "Hello"
+                .WithRenderEngine(RazorTemplateRendererProvider.ProviderName)
```

Comment:
What about to define some sugar extension methods like `.WithRazorTemplate()` and `.WithScribanTemplate()` as shortcut of `.WithRenderEngine` (keep it for extensibility).